components/item-act.js

```diff
@@ -301,10 +242,10 @@ export class ActCanceledError extends Error {
 }
 
 export class ZapUndoController extends AbortController {
-  constructor () {
+  constructor ({ onStart = () => {}, onDone = () => {} }) {
     super()
-    this.signal.start = () => { this.started = true }
-    this.signal.done = () => { this.done = true }
+    this.signal.start = onStart
+    this.signal.done = onDone
```

Comment:
This was coincidentally updating `UpVote`s state and causing it to rerender previously. Here, we make it explicitly update `UpVote`s state.